### DISCUSSION TITLE
neomutt: profile creates a bunch of empty useless files

### DISCUSSION THREAD

**Discussion by WhyNotHugo** [netblue30/firejail#5276] [Q&A] [answered]
The profile for neomutt creates twenty five file and directories, most of which are not needed. Most of these are entirely useless by being empty too:

https://github.com/netblue30/firejail/blob/master/etc/profile-m-z/neomutt.profile#L52-L77

In particular, all the `mkfile` entries seem to be useless:

```
mkfile ${HOME}/.emacs
mkfile ${HOME}/.mailcap
mkfile ${HOME}/.msmtprc
mkfile ${HOME}/.muttrc
mkfile ${HOME}/.nanorc
mkfile ${HOME}/.neomuttrc
mkfile ${HOME}/.signature
mkfile ${HOME}/.viminfo
mkfile ${HOME}/.vimrc
```

neomutt won't write to these locations. Processes it spawns might _read_ to some of them, but creating an empty file doesn't help. It also seems to create empty directories for emac's configuration files and a for a few other applications.

Can we drop all this creation to the bare minimum? If a user uses emacs, then they'll have a configuration directory already. Creating an empty one doesn't help; they still have to populate it. Having the neomutt profile create such a directory is just noise for everyone else (also, a large amount of these are in `$HOME`).

**Comment by rusty-snake** [comment, marked as answer]
`mkfile`/`mkdir` for editor configurations can be removed IMO.

**Comment by kmk3** [comment]
For reference, this should be fixed by:

* #5278
